<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>preferences</class>
 <widget class="QDialog" name="preferences">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>454</width>
    <height>366</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Préférences</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>411</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>414</width>
     <height>276</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="1">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="12" column="1">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Taille des blocs</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Niveau de départ</string>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QSpinBox" name="plateauColonnes">
      <property name="minimum">
       <number>10</number>
      </property>
      <property name="maximum">
       <number>999</number>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="12" column="2">
     <widget class="QSpinBox" name="themeTaille">
      <property name="minimum">
       <number>10</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Nombre de cases verticales</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="6" column="1">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Nombre de cases horizontales</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QSpinBox" name="jeuLevel">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>16</number>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QComboBox" name="jeuMode">
      <item>
       <property name="text">
        <string>Hexagonal</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Traditionnel</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="11" column="1">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Choix du thème graphique</string>
      </property>
     </widget>
    </item>
    <item row="11" column="2">
     <widget class="QComboBox" name="themeName"/>
    </item>
    <item row="7" column="2">
     <widget class="QSpinBox" name="plateauLignes">
      <property name="minimum">
       <number>20</number>
      </property>
      <property name="maximum">
       <number>999</number>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Mode de jeu</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="label_6">
      <property name="font">
       <font>
        <family>Noto Sans</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Modalités de jeu</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="QLabel" name="label_5">
      <property name="font">
       <font>
        <family>Noto Sans</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Plateau</string>
      </property>
     </widget>
    </item>
    <item row="10" column="0" colspan="2">
     <widget class="QLabel" name="label_9">
      <property name="font">
       <font>
        <family>Noto Sans</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Thème</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>preferences</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>preferences</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
